carbon-design-system/carbon
# chore(react): remove deprecated stories from storybook

Removes the deprecated `ToggleSmall` and `Toolbar` code to remove the stories from our Storybook

#### Changelog

**New**

**Changed**

**Removed**

- ToggleSmall and Toolbar

#### Testing / Reviewing

- Verify there is no deprecation section on the site
- Verify ToggleSmall/Toolbar are not visible in the side nav
